main.go

```diff
@@ -77,6 +78,9 @@ func main() {
 	opts := zap.Options{
 		Development: true,
 	}
+	flag.BoolVar(&backgroundMode, "background-mode", true,
+		"Enable PolicyException background mode.",
```

Comment:
```suggestion
	flag.BoolVar(&backgroundMode, "background-mode", false,
		"Enable PolicyException background mode. If true, failing resources have a status of 'skip' in reports, instead of 'fail'. Defaults to false.",
```
Personally I'd make `false` the default. Follows upstream and is what we want long-term